easybuild/tools/options.py

```diff
@@ -258,7 +258,7 @@ def override_options(self):
                                             None, 'store_true', False),
             'cleanup-builddir': ("Cleanup build dir after successful installation.", None, 'store_true', True),
             'cleanup-tmpdir': ("Cleanup tmp dir after successful run.", None, 'store_true', True),
-            'color': ("Allow color output", None, 'store_true', True),
+            'color': ("Colorize output", 'choice', 'store', 'auto', ['auto', 'always', 'never'], {'metavar':'WHEN'}),
```

Comment:
>> also, changing this may break people's configuration files, since they
>> may specify `False` or `True` now, so we'll need to be careful with
>> changing this to string, and at least translate `False` to `never` and
>> `True` to `auto`
>
> Good point, but how to I do that with vsc's option parsing framework?

Actually, I would translate the other way round:

* `True` -> `auto`
* `False` -> `never`